package.json

```diff
@@ -46,7 +45,7 @@
     "@types/superagent": "2.0.35",
     "@types/winston": "2.3.9",
     "@types/ws": "0.0.37",
-    "bignumber.js": "4.0.2",
+    "bignumber.js": "5.0.0",
```

Comment:
Why not upgrade to version 6?

Reply:
It's got some changes I don't have the time to figure out yet, such as removing `round()`: https://github.com/MikeMcl/bignumber.js/blob/master/CHANGELOG.md